iina/Utility.swift

```diff
@@ -343,6 +343,12 @@ class Utility {
     return url
   }()
 
+  static let thumbnailCacheURL: URL = {
+    let url = Utility.appSupportDirUrl.appendingPathComponent(AppData.thumbnailCacheFolder, isDirectory: true)
```

Comment:
Generating thumbnails can be costly, like 10-20s for large videos, so we may want to save them permanently. Of course I think we should also provide a button in Preferences to clear all caches.